docs/reference/bodypose.md

```diff
@@ -240,12 +240,52 @@ let bodypose = ml5.bodypose(?model, ?options, ?callback);
 
 - **options**: Optional. An object to change the default configuration of the model. The available options differ depending on which of the two underlying models are used.
 
-See See the [MoveNet documentation](https://github.com/tensorflow/tfjs-models/tree/master/pose-detection/src/movenet#create-a-detector) and the [BlazePose documentation](https://github.com/tensorflow/tfjs-models/tree/master/pose-detection/src/blazepose_tfjs#create-a-detector) for more information on available options.
+  The default and available options are:
 
-- **callback(bodypose, error)**: Optional. A "callback" function that runs when the model has been successfully loaded. Most ml5.js example call `ml5.bodyPose()` in the p5.js `preload()` function and no callback is needed.
+  ```javascript
+  {
+    modelType: "MULTIPOSE_LIGHTNING" // "MULTIPOSE_LIGHTNING", "SINGLEPOSE_LIGHTNING", or "SINGLEPOSE_THUNDE"
+    enableSmoothing: true,
+    minPoseScore: 0.25,
+    multiPoseMaxDimension: 256,
+    enableTracking: true,
+    trackerType: "boundingBox", // "keypoint" or "boundingBox"
+    trackerConfig: {},
+    modelUrl: undefined,
+  }
+  ```
+
+  Options for both models:
+  - _modelType_ - Optional
+    - String: The type of model to use. Default: "MULTIPOSE_LIGHTNING".
+  - _enableSmoothing_ - Optional
+    - Boolean: Whether to smooth the pose landmarks across different input images to reduce jitter. Default: true.
+
```

Comment:
We may need to add `Options for MoveNet model only` as well. See https://github.com/ml5js/ml5-next-gen/blob/main/src/BodyPose/index.js#L27-L32 for reference

Reply:
Thanks for point that out! It was in the previous commit but I accidentally removed it when solving conflict when updating. Will fix it now.